src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -97,43 +105,14 @@ internal class LaunchSettingsProvider : OnceInitializedOnceDisposed, ILaunchSett
 
         private SequencialTaskExecutor _sequentialTaskQueue = new SequencialTaskExecutor();
 
-        /// <summary>
-        /// Returns the full path to the launch settings file
-        /// </summary>
-        private string _launchSettingsFile;
+        [Obsolete("Use GetLaunchSettingsFileAsync instead.")]
```

Comment:
@BillHiebert I didn't, I can't find the usage of it so wanted to avoid making a breaking change. Do you know who consumes this?